### DISCUSSION TITLE
wiki.js automatic setup not working

### DISCUSSION THREAD

**Discussion by lorneeaston** [requarks/wiki#7171] [Error / Bug Report]
I am following the instructions here:

https://github.com/requarks/wiki-v1/issues/45

I am deploying requarks/wiki:2 to an Azure Linux Web App.

I want to configure the instance using automation. Despite WIKI_JS_HEROKU and WIKI_ADMIN_EMAIL being set, I still get the splash screen to complete the install.

My goal is to configure using https://registry.terraform.io/providers/startnext/wikijs/latest/docs/resources/api

**Comment by lorneeaston** [comment]
I am able to successfully interact with the API using startnext/wikijs, and enable the APIs, but only after manually completing setup and creating an admin user. So automatic setup is clearly not working in this case.

**Reply by benjamsf** [reply]
I kind of have the same problem. 

Moreover, there is no way to validate config what you are trying to use. If there's errors in the config file you're trying to use, it'll just overlook it (telltale sign is that it runs the GUI setup at port 80 even if your config.yml says SERVER_PORT: 3000).

Once your config appears to be valid enough, it'll run the GUI setup at SERVER_PORT: whatever you have. But nothing makes the config valid enough to go on without the GUI setup. It always says "DB setup incomplete..."... is unattended install actually something you just can't do?
